File: UNIT 3/assessment/4.py
n = int(input("Input = "))
print("Output =")

for i in range(0, n, 2):
    for j in range(0, i + 1, 2):
        if j % 4 == 0:
            print(j, end="")
        else:
            print("#", end="")
    print()

File: UNIT 3/assessment/4.c
#include <stdio.h>

int main() {
    int n, i, j;
    printf("Input = ");
    scanf("%d", &n);
    printf("Output =\n");

    for (i = 0; i < n; i += 2) {
        for (j = 0; j <= i; j += 2) {
            if (j % 4 == 0)
                printf("%d", j);
            else
                printf("#");
        }
        printf("\n");
    }
    return 0;
}
